content/_data/changelogs/lts.yml

```
@@ -8513,6 +8513,414 @@
+
+  - type: major rfe
+    category: rfe
+    pull: 6843
+    authors:
+    pr_title: "Hide potentially sensitive values (system properties and environment variables) by default"
+    references:
+      - pull: 6843
+      - pull: 6995
+      - issue: 69339
+      - pull: 7373
+      - pull: 7452
+      - pull: 7173
+      - issue: 70128
+      - pull: 7556
+      - issue: 70112
+      - pull: 7555
+      - pull: 7475
+      - issue: 70209
+      - pull: 7425
+      - issue: 70117
+      - pull: 7597
+      - issue: 70240
+      - pull: 7314
+      - pull: 7203
+      - pull: 7171
+      - pull: 7364
+      - issue: 69517
+      - pull: 7366
+      - pull: 7364
+      - pull: 7229
+      - issue: 69715
+      - pull: 7352
+      - pull: 7367
+      - pull: 7368
+      - pull: 7399
+      - issue: 70036
+      - pull: 7427
+      - issue: 70121
+      - pull: 6511
+    message: |-
+      Update to various UI elements. 
+      Modernize table display, page layout, and buttons.
+  - type: major rfe
+    category: major rfe
+    pull: 7457
+    issue: 70169
+    authors:
+      - lemeurherve
+      - timja
+    pr_title: "[JENKINS-70169] Add missing breadcrumb items"
+    references:
+      - pull: 6912
+      - pull: 7487
+      - pull: 7488
+      - pull: 7489
+      - pull: 7490
+      - pull: 7491
+      - pull: 7492
+      - pull: 7493
+      - pull: 7494
+      - pull: 7495
+      - pull: 7496
+    message: |-
+      Add missing breadcrumb items in various locations.
+  - type: major rfe
+    category: major rfe
+    pull: 7293
+    authors:
+      - slide
+    pr_title: "[JENKINS-68652] Update ANTLR2 to ANTLR4"
+    message: |-
+      Update ANTLR2 grammars and code to ANTLR4.
+  - type: major rfe
+    category: major rfe
+    pull: 7436
+    authors:
+      - dependabot
+    pr_title: "Upgrade Spring Security from 5.7.4 to 5.8.0"
+    references:
+      - url: https://github.com/spring-projects/spring-security/releases/tag/5.7.5
+        title: Spring Security 5.7.5 release notes
+      - url: https://github.com/spring-projects/spring-security/releases/tag/5.8.0
+        title: Spring Security 5.8.0 release notes
+      - url: https://spring.io/security/cve-2022-31690
+        title: CVE-2022-31690
+      - url: https://spring.io/security/cve-2022-31692
+        title: CVE-2022-31692
+    message: |-
+      Update Spring Security from 5.7.4 to 5.8.0.
+      This update includes several fixes and improvements.
+  - type: major rfe
+    category: major rfe
+    pull: 7376
+    issue: 64151
+    authors:
+      - slide
+    pr_title: "[JENKINS-64151] Set a default file size rotation of AsyncPeriodicWork…"
+    message: |-
+      Set default file size rotation of AsyncPeriodicWork / AsyncAperiodicWork task logs.
+  - type: major rfe
+    category: rfe
+    pull: 6408
+    authors:
+      - janfaracik
+    pr_title: "Replace YUI tooltips with Tippy.js"
+    message: |-
+      Update appearance and framework for tooltips.
```

```suggestion
      Update appearance of tooltips and replace old library with Tippy.js.
```
I think we should state what we are using now. There's a great chance someone has already heard of Tippy.js before.